### PR TITLE
refactor(linter): UseAnchorContent code action

### PR DESCRIPTION
**Pull request by vasucp1207**
## Summary

Closes https://github.com/biomejs/biome/issues/362

### CODE REVIEW

File: crates/biome_js_analyze/src/analyzers/a11y/use_anchor_content.rs [hunk 1]
```diff
@@ -117,8 +120,32 @@ impl Rule for UseAnchorContent {
             markup! {
                 "All links on a page should have content that is accessible to screen readers."
             }
+        ).note(
+            markup! {
+                "Accessible content refers to digital content that is designed and structured in a way that makes it easy for people with disabilities to access, understand, and interact with using assistive technologies. Follow the links for more information,
+                "<Hyperlink href="https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context">"WCAG 2.4.4"</Hyperlink>"
+                "<Hyperlink href="https://www.w3.org/WAI/WCAG21/Understanding/name-role-value">"WCAG 4.1.2"</Hyperlink>""
```
Comment (ematipico) [inline crates/biome_js_analyze/src/analyzers/a11y/use_anchor_content.rs:127]: It seems that this text goes on a new line with a lot of space. Is it intended? 
Screenshot 2023-10-06 at 08 51 32

Comment (vasucp1207) [reply crates/biome_js_analyze/src/analyzers/a11y/use_anchor_content.rs:127]: Oh, forgot the linebreaks.

**Comment by vasucp1207** [comment]
Hm!, don't know why ci is failing even after running `just codegen-linter` and `just codegen-website`.

**Comment by ematipico** [comment]
I think I know why. I think you have to rebase your `main` with `upstream/main`, then run the codegen again and maybe it will work

**Review by ematipico — APPROVED** [review]

**Pull request merged**